README.markdown

```diff
@@ -165,16 +164,16 @@ func swap<Thing>(_ a: inout Thing, _ b: inout Thing)
 
 ### Language
 
-Use US English spelling to match Apple's API.
+Use UK English spelling to match Apple's API.
```

Comment:
I believe this is not correct, must confirm with Fabio, but I do believe spelling should be US to be consistent with international and Apple coding de-facto "standards".